src/Command/DatabaseCommand.php

```diff
@@ -50,6 +50,12 @@ public static function fromGlobal(GlobalConfig $global, string $fromInstanceName
             }
             $result[] = static::fromAppInstances($fromInstance, $toInstance, $fromDatabase, $toDatabase, $currentHost, $all);
         }
+        foreach ($fromInstance->getDatabases() as $databaseName => $fromDatabase) {
+            if ($toInstance->hasDatabase($databaseName)) {
```

Comment:
i stumbled upon this while reading: i know toInstance is meant as its the instance where things are going to but while reading it, "toInstance" sounds like you made / make something to an instance.. I'm aware this is not really an issue but i just wanted to point this out

Reply:
``InstanceTo``? is worse

Reply:
alternatively: source, destination - but ofc thats longer